renderdoc/driver/d3d12/d3d12_manager.cpp

```diff
@@ -373,12 +373,8 @@ bool D3D12ResourceManager::Prepare_InitialState(ID3D12DeviceChild *res)
 
     if(desc.Dimension == D3D12_RESOURCE_DIMENSION_BUFFER)
     {
-      D3D12_HEAP_PROPERTIES heapProps;
-      heapProps.Type = D3D12_HEAP_TYPE_READBACK;
-      heapProps.CPUPageProperty = D3D12_CPU_PAGE_PROPERTY_UNKNOWN;
-      heapProps.MemoryPoolPreference = D3D12_MEMORY_POOL_UNKNOWN;
-      heapProps.CreationNodeMask = 1;
-      heapProps.VisibleNodeMask = 1;
+      D3D12_HEAP_PROPERTIES heapProps =
+          m_Device->GetReal()->GetCustomHeapProperties(1, D3D12_HEAP_TYPE_READBACK);
```

Comment:
What's the purpose of this change? My understanding is that `GetCustomHeapProperties` returns an entirely equivalent set of heap properties to using the `READBACK` macro.